internal/mail/message_test.go

```diff
@@ -71,38 +133,38 @@ func TestNewMessage(t *testing.T) {
 
 func Test_detectContentType(t *testing.T) {
```

Comment:
Can you rename this test function to `Test_contentTypeOrDefault`

Reply:
Done!